src/apiCalls.js

```diff
@@ -14,3 +14,8 @@ export const submitLoginCredentials = (state) => {
   })
   .then(response => response.json())
```

Comment:
There's an opportunity for more error handling here, see a comment I made about response.ok